spec.md

```diff
@@ -1762,11 +1762,10 @@ message NodeStageVolumeRequest {
 
   // The path to which the volume MAY be staged. It MUST be an
   // absolute path in the root filesystem of the process serving this
-  // request. The CO SHALL ensure that there is only one
-  // `staging_target_path` per volume. The CO SHALL ensure that the
-  // process serving the request has `read` and `write` permission to
-  // the path, and is able to create files or directories at the path
-  // if it does not exist.
+  // request, and MUST be a directory. The CO SHALL ensure that there
+  // is only one `staging_target_path` per volume. The CO SHALL ensure
```

Comment:
multiple ways to interpret: "... path is an existing directory". Suggest re-phrasing:

The CO SHALL be responsible for creating the path if it does not exist. The CO SHALL ensure that the path is a directory.

Reply:
Thanks, I will use your language here.